package.json

```diff
@@ -34,8 +34,9 @@
     "tailwind.config.*"
   ],
   "scripts": {
-    "build": "yarn build:clean && yarn build:tokens && yarn build:js && yarn copy-fonts-to-lib",
+    "build": "yarn build:clean && yarn build:tokens && yarn build:declarations && yarn build:js && yarn copy-fonts-to-lib",
     "build:clean": "rm -rf lib/",
+    "build:declarations": "tsc --emitDeclarationOnly --project tsconfig.build.json",
```

Comment:
Does this also work with `--noEmit` be added to`build:js` portion?

Reply:
Rollup used in `build:js` also relies on `tsconfig.build.json` and having `noEmit` in there causes a logical error with building declarations `TS5053: Option 'emitDeclarationOnly' cannot be specified with option 'noEmit'.`